aea/protocols/generator.py

```diff
@@ -161,56 +168,71 @@ def _message_class_str(self) -> str:
         cls_str += '    def __init__(self, message_id: int, target: int, performative: str, contents: Dict, **kwargs):\n'
         cls_str += '        \"\"\"Initialise.\"\"\"\n'
         cls_str += '        super().__init__(message_id=message_id, target=target, performative=performative, contents=contents, **kwargs)\n\n'
-
-        # variables
-        # cls_str += str.format('        self.performatives = {}\n', self._performatives_set())
-        cls_str += str.format('        self.speech_acts = {}\n\n', self._speech_acts_str())
+        cls_str += str.format('        self.speech_acts_definition = {}\n\n', self._speech_acts_str())
```

Comment:
Can we simply call this `speech_acts`? The definition bit does not add anything for me.

Reply:
Done